webhooks-extension/docs/GettingStarted.md

```diff
@@ -0,0 +1,161 @@
+
+
+## Creating a new webhook
+
+The Tekton dashboard webhooks extension creates an association between a git repository and a Tekton pipeline. Changes to a branch or pull request in the associated git repository will cause a new PipelineRun and PipelineResource objects to be created. This section covers the various setup and configuration options in more detail. The image below shows the current form of the main 'Create Webhook' panel in the dashboard. 
+
+![Creating a new webhook](screenShot1.png?raw=true "Creating a new webhook")
+
+### Name
+
+A descriptive name for this webhook. Names must currently be unique; we aim to relax this restriction. 
+
+### Repository URL
+
+The URL of the git repository in which the webhook should be created. 
+
+### Access Token
+
+A GitHub access token that will be used to create the webhook. You may have created an access token in the section  'Create credentials: Git' above. If you chose to have one access token for both checking out code and creating webhooks then you can reuse its value here. 
+
+1. If you did not do so earlier, create an access token via GitHub. As of August 2019, for a given GitHub repository, visit [your name icon in the top right corner] > Settings > Developer settings > Personal access tokens > Generate new token and create a token with `admin:repo_hook` permissions. 
+2. Access tokens for webhooks are stored differently to credentials for Tekton Pipelines. You must store an access token via the 'Access Token' menu even if you created an access token-based Git secret earlier. 
+
+### Namespace
+
+Select the kubernetes namespace in which the triggered Pipeline should run. 
+
+### Pipeline
+
+Select the Pipeline in the target Namespace from which a PipelineRun (and accompanying PipelineResources) will be created, as described in the section 'What does the Tekton Dashboard's Webhooks Extension do?', above. 
+
+### Service Account
+
+This field specifies the service account that will be used by the PipelineRun. It should be the same service account that you set up RBAC permissions for in the section 'Setting up a webhook from scratch' and that you patched credentials onto in the sections, 'Create credentials: Git' and 'Create credentials: Docker', above. 
+
+### Docker Registry
+
+Finally, select the Docker registry that any built images should be pushed to. This is the docker registry that the 'docker' credentials earlier are associated with. 
+
+## Putting it all together: test it's working
+
+Once a webhook is set up, a `git push` or merged Pull Request to a branch in the monitored repository should trigger the creation of a correct PipelineRun. This PipelineRun will show up in the Tekton dashboard as usual. As of August 2019, a successful webhook will trigger pod changes in the target namespace of the form shown below. 
+
+Firstly, webhook receipt will drive a pod whose name corresponds to the 'Name' field of the webhook. E.g. for a Name of `buildah-hook` we see, 
```

Comment:
It'll `create` a pod, to me `drive` implies giving direction to a pod implying that it already exists so users may be expecting to see the below pod when it doesn't yet exist